frontend/src/components/Editor/Editor.js

```diff
@@ -127,9 +121,10 @@ const Editor = ({
     if (fileName === "indexingLogic.js") {
       try {
         setSchemaTypes(pgSchemaTypeGen.generateTypes(schema));
-        setError(() => undefined);
-      } catch (error) {
-        handleCodeGenError(error);
+        setTypesGenerationError(() => undefined);
```

Comment:
```suggestion
        setTypesGenerationError(undefined);
```
Using a function is only necessary when you intend to use the existing state, i.e. `(currentState) => currentState + 1`